.asset-template.json.hb

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Block Coding",
+  "description": "Create games using a high-level, block-based visual programming language.\r\n\r\nIntended as an educational tool for learners in the earlier stages of their journey towards becoming game developers. This plugin lets you create your first games with high-level blocks, avoiding the immediate need to learn to code in GDScript. Building games in this way provides a gentle introduction to programming concepts and allows you to focus your efforts on becoming familiar with the rest of the Godot Editor UI.",
+  "category_id": "5",
+  "godot_version": "4.2",
+  "version_string": "{{ context.release.tag_name }}",
+  "cost": "MIT",
+  "support_level": "community",
+  "download_provider": "GitHub",
+  "download_commit": "{{ env.GITHUB_SHA }}",
+  "browse_url": "{{ context.repository.html_url }}",
+  "issues_url": "{{ context.repository.html_url }}/issues",
+  "icon_url": "https://raw.githubusercontent.com/endlessm/godot-block-coding/{{ context.release.tag_name }}/icon.png",
```

Comment:
Probably a smart thing to do so that twiddling with the icon on `main` won't change the asset until we make a release.